platforms/gradle/foundry-gradle-plugin/src/main/kotlin/foundry/gradle/FoundryProperties.kt

```diff
@@ -459,6 +459,26 @@ internal constructor(
   public val useOrchestrator: Boolean
     get() = booleanProperty("foundry.android.test.orchestrator")
 
+  /**
+   * Flag for compressing androidTest APks with legacy packaging.
+   *
+   * See:
+   * - https://issuetracker.google.com/issues/259832799
+   * - https://developer.android.com/reference/tools/gradle-api/7.1/com/android/build/api/dsl/DexPackagingOptions#useLegacyPackaging:kotlin.Boolean
+   */
+  public val compressAndroidTestApksWithLegacyPackaging: Provider<Boolean>
+    get() = resolver.booleanProvider("foundry.android.test.compressWithLegacyPackaging", false)
+
+  /**
+   * Flag for minifying androidTest APks with R8. This just tree shakes.
+   *
+   * See:
+   * - https://issuetracker.google.com/issues/259832799
+   * - https://developer.android.com/reference/tools/gradle-api/7.1/com/android/build/api/dsl/DexPackagingOptions#useLegacyPackaging:kotlin.Boolean
+   */
+  public val minifyAndroidTestApks: Provider<Boolean>
```

Comment:
could we rename this to `compressAndroidTestApks`. I'm concerned that minify will get conflated with running shrinking.

Reply:
We already have a compress property above this, this is the toe-hold for minification with R8 if there ends up being a way to do that too